src/script/api/script_text.cpp

```diff
@@ -101,7 +101,6 @@ SQInteger ScriptText::_SetParam(int parameter, HSQUIRRELVM vm)
 		default: return SQ_ERROR;
```

Comment:
Since empty params are now possible, I'd add support for them.
```suggestion
		case OT_NULL:
			this->param[parameter] = {};
			break;

		default: return SQ_ERROR;
```